src/main/resources/templates/macros/report/elements.vm

```diff
@@ -39,6 +39,8 @@
                     </tr>
                   #end
                 </table>
+              #elseif ($step.getDocString())
```

Comment:
why elseif? I guess rows and docstring can exist together so there should be simple if, not elseif